arcsi/api/show.py

```diff
@@ -94,8 +109,8 @@ def list_shows():
     return shows_schedule_schema.dumps(shows)
 
 
-@arcsi.route("/show/archive", methods=["GET"])
-def list_shows_archive():
+@arcsi.route("/show/all_page", methods=["GET"])
```

Comment:
Why not use this function body of this method as `/show` && `/show/all`'s body?

Reply:
You are right, I defined a new `get_items()` method in the utils.py.